.circleci/config.yml

```diff
@@ -1,700 +1,655 @@
+---
 version: 2.1
 
 orbs:
   # https://circleci.com/orbs/registry/orb/circleci/slack
   slack: circleci/slack@3.4.2
 
-# ======================================================================================================================
-
-# Executors define the environment in which the steps of a job will be run, allowing you to reuse a single executor 
-# definition across multiple jobs. https://circleci.com/docs/2.0/configuration-reference/#executors-requires-version-21
-executors:
-    xcode-executor:
-        parameters:
-            xcode:
-                type: string
-                default: "12.2.0"
-        macos:
-            xcode: << parameters.xcode >>
-        environment:
-            HOMEBREW_NO_AUTO_UPDATE: 1
-            HOMEBREW_NO_INSTALL_CLEANUP: 1
-
-# ======================================================================================================================
```

Comment:
I don't think we should remove this - it has been useful in the past to test against multiple versions of Xcode using the matrix functionality (e.g. when we want to test with current stable and beta versions).

Reply:
okay, will revert

Reply:
[This](https://github.com/mapbox/mapbox-maps-ios/pull/175/commits/d42c11eda4583684c2d2771cfbe00ca900c45037) should be sufficient, yeah?

Reply:
Yep, and re-add the executor section.

Reply:
Oh I see. Yeah, let's keep that.